### PR TITLE
Start the bridge as a process in the release gate too, not only the PR canary

### PR DESCRIPTION
**Pull request by davekilleen**
Closes the last open item from #458. Announced on the card before building, since
unannounced pickup is what produced the #457/#458 duplicate.

## The gap

#458 made the **PR canary** launch the published bridge as a top-level process.
The **formal fleet** — the gate that runs before a release is published — still
called `run_journey()` without it. So a release could be published having never
once started the bridge the way a stuck user starts it: the environment scrub,
the `execve` into the vault virtualenv, the clean-runtime check. Same class as
the original card, one level up, on the gate that matters more.

## What changes

`collect_platform_runs` passes `bridge_process_entry=True` for exactly one
release, chosen by `bridge_process_entry_release()`.

## On your correlation question — yes, it was a real gap, and no, I did not inherit the canary's choice

You asked whether both gates now pick the same release. They would have, and it
would have mattered.

The formal fleet picks the **newest** release in the cohort; the canary picks the
**oldest** of its fixed starts. The two vault shapes take *different paths
through the bridge*:

- a pre-split vault stops first at the **topology** approval — *"Dex will first
  separate its code from your notes."*
- an already brain/vault-split vault skips that entirely (`run_bridge` records
  `already-brain-vault-split` and asks for no token), so its first gate is the
  **delivered-release** approval.

This is not theoretical: the canary transcript I read yesterday on a `v1.81.11`
vault ends on `Dex will now install verified foundation v1.81.16. Type APPLY to
continue:` — the second gate, never the first.

Two gates both probing the oldest start would be correlated and would miss the
same thing: **no gate would ever watch the bridge reach the delivered-release
gate as a real process.** One function owns the choice, so it is one place to
change if the reasoning does.

## The floor, and watching it fail

`_assert_bridge_process_entry_ran` reads the evidence directory the probe leaves
behind and requires exactly one, belonging to the designated release. It checks
the **artifact, not the intention** — the probe is wired in by a single boolean.

Per your condition, demonstrated against the real wiring rather than a fake:

```
STEP 1 — green with the wiring intact:          1 passed
STEP 2 — bridge_process_entry replaced by False:
  PlatformFleetFailure: no release started the bridge as a top-level process:
  expected evidence at .../v1.80.5-222222222222.bridge-process-entry/bridge-process-entry.json
                                                1 failed
STEP 3 — restored:                              1 passed
```

A previously green platform run turns red purely because the probe did not run.
Encoded permanently as two tests: a probe that never ran, and evidence belonging
to the wrong release.

## ⚠️ What is proven, and what is not

**Proven:** the wiring, the choice of release, the floor firing and clearing,
and that the stand-in journey runners now produce the artifact they stand for.
256 tests pass locally; the 3 failures in
`test_release_fleet_acceptance.py` are the known pre-existing macOS-only ones,
identical to `main`'s baseline.

**NOT proven:** the live formal fleet. It is a manual `workflow_dispatch`,
macOS-only, ~6 hours, and requires a published follow-up release — I cannot run
it here, and no CI job on this PR exercises it.

**What would prove it:** one manual dispatch of `historic-fleet-darwin` (formal
mode) that someone watches, confirming the run completes and that exactly one
`*.bridge-process-entry/` directory appears in the retained evidence, belonging
to the newest release in the cohort. Until that happens this is honest partial
verification, and I would rather say so than let it read as complete.

Note the PR canary does **not** cover this change: `release_fleet_acceptance.py`
is in the canary's path filter, so the canary runs, but the canary exercises
`run_canary`, not the formal path. Its green tells you nothing about this wiring.

## Not needed

No journey-protocol regeneration — the protocol pins `release_fleet.py`,
`release_fleet_executor.py` and `dex_update_bridge.py`, none of which this
touches.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Medium Risk**
> Changes release-gate orchestration in `release_fleet_acceptance.py`; mis-selection or a broken evidence floor could block publishes or give false confidence, though behavior is heavily tested and fail-closed.
> 
> **Overview**
> Extends **formal historic fleet acceptance** so it mirrors the PR canary’s “start the bridge as a process” check, closing the gap where a release could pass the publish gate without ever exercising real user entry.
> 
> **`bridge_process_entry_release()`** picks exactly one cohort start: the **newest** historic release that is **not** the pinned foundation (tag/commit/tree), deliberately **not** the canary’s oldest start, so the two gates cover different vault shapes (topology vs delivered-release approvals).
> 
> **`collect_platform_runs`** passes `bridge_process_entry=True` only for that release. **`_assert_bridge_process_entry_ran`** requires exactly one `bridge-process-entry.json` under the designated case’s evidence directory after all journeys succeed—checking artifacts, not intent.
> 
> Docs in `docs/release-fleet-acceptance.md` explain the rationale. Tests cover selection edge cases (foundation-only cohort, wrong-release evidence, silent probe) and update existing harnesses to write stand-in evidence when probed.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit b330e0c3ecd0aa1856f7d0c8dd0af5ad0b29e849. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Comment by davekilleen** [comment]
Lint fixed and pushed (`a6613b89`). `ruff check core/` — the exact CI scope — now passes; 43 acceptance tests pass with the same 3 pre-existing macOS-only failures as `main`.

The cause is worth one line, because it is a small instance of the thing this PR is about: the new helper annotated a `Sequence` parameter without importing it. `from __future__ import annotations` means the annotation is never evaluated at runtime, so **the tests passed** — and I let that stand as my signal despite noticing the missing import earlier. Ruff's F821 was right and my green was not evidence. Runtime-passing is not lint-passing.

**Re your condition (2) — yes, both are already in the PR body, near the top, and I will stop repeating them here:**

- The floor demonstrated failing against the real wiring is under *"The floor, and watching it fail"*: green with wiring intact → `bridge_process_entry` replaced by a literal `False` → `PlatformFleetFailure: no release started the bridge as a top-level process` → restored → green. Plus two permanent tests (probe never ran; evidence from the wrong release).
- The unproven-live caveat is its own **⚠️ What is proven, and what is not** section: wiring, release choice and floor are proven; the live formal fleet is not, because it is a manual macOS-only ~6-hour dispatch. I named what would settle it — one watched `historic-fleet-darwin` formal dispatch confirming exactly one `*.bridge-process-entry/` directory appears, belonging to the newest release in the cohort. I also flagged that the PR canary running here exercises `run_canary`, not the formal path, so its green says nothing about this change.

**Unrelated but you should know, since you mentioned machine load:** this box hit `ENOSPC` twice while I was working — `/tmp` is a 47G **tmpfs**, so it is RAM. `/tmp/pytest-of-dexdev` alone held **19G** of accumulated pytest scratch from many sessions. I removed my own leftovers (a downloaded CI artifact, git clones, two worktrees I had registered) and trimmed pytest scratch older than two hours, taking `/tmp` from full to 26G free. Nothing was lost, but that directory will refill: nothing prunes it, and every session's `tmp_path` fixtures land there. Worth a janitor entry rather than each session discovering it at `ENOSPC`.

**Comment by davekilleen** [comment]
Independent review after folding in current Dex: **ready to merge.**

The full release check now starts the rescue helper as a real process for exactly one starting version — the newest old install in the set, not the same oldest one the shorter check already covers. The safety floor looks at leftover proof on disk, not at the intention to run. Missing proof, or proof for the wrong version, fails the check. The tests that stand in for a real run actually write that leftover file.

What this does **not** claim: the six-hour live all-versions Mac job has not been run on this change. That remains an honest gap, not a silent miss. The wiring cannot quietly turn itself off; if the helper never starts, the check goes red.

GitHub will not accept a self-approval on this pull request. Required checks are `quality` and `test-results`. Merging once those are green on this refresh.

File: scripts/release_fleet_acceptance.py [hunk 1]
```diff
@@ -699,6 +699,72 @@ def _write_platform_failure_summary(
...
+    is already brain/vault split skips that entirely (``run_bridge`` records
+    ``already-brain-vault-split`` and asks for no token), so its first approval
+    gate is the delivered-release one. Probing only the oldest start would mean
+    no gate ever watched the bridge reach the second of those as a real process.
+
+    ``discover_distribution_releases`` sorts ascending by semantic version, so
+    the newest is the last entry.
+    """
+
+    return releases[-1] if releases else None
```
[the remark this answers is not in the document]
Comment (davekilleen) [reply scripts/release_fleet_acceptance.py]: Confirmed, then narrowed.

Live public historic set today has 202 starts. Newest is semantic `v1.81.16` (commit `fcf5a23`), not the pinned foundation `dist/release/v1.81.16-281202d` (`281202d`). Installing that sibling does **not** set `refs/dex/installed` to the pin, so `run_bridge` still reaches the delivered-release `APPLY`. The current-head probe would have worked by coincidence.

The hole is still real. The cohort is required to contain the exact pin. When that identity is last — no sibling tree, or the foundation later rotates to a version with only the dist tag — `releases[-1]` is the start whose installer already matches the pin. Then `_foundation_is_installed` is true and both approvals are skipped. The process probe would either fail closed (no `APPLY`) or pass on an MCP `APPLY` it was not meant to watch.

`528c84f7` excludes the exact foundation identity (tag, commit, tree). A distinct same-version sibling stays eligible. An only-foundation set now has no probe and the existing floor fails closed. Tests cover: pin is last; sibling is last; only-foundation; collector requests the previous start; the silent-no-probe floor is unchanged.

**Comment by davekilleen** [comment]
Independent review after the foundation-exclusion follow-up: **ready to merge.**

The release check now skips the exact pinned first-hop install when it starts the rescue helper as a real program. That start already has the installed marker, so the helper would skip both confirmation screens. A same-version sibling stays eligible — that is today's live newest old install. If the only start left is the pin, the check fails closed and now says so instead of claiming the set was empty.

The live public historic set is locked in a test: the chosen start exists, is not the pin, and is not the oldest start the shorter check already covers.

Required GitHub checks are still `quality` and `test-results`. Merging when those are green on this head (`b330e0c3`).

Still not claimed: the six-hour live all-versions Mac job has not been run on this change.
